src/main/kotlin/no/nav/familie/ef/sak/service/VurderingStegService.kt

```diff
@@ -95,9 +104,27 @@ class VurderingStegService(private val behandlingService: BehandlingService,
             stegService.håndterVilkår(behandling).id
         } else if (behandling.steg != StegType.VILKÅR && vilkårsresultat.any { it == Vilkårsresultat.IKKE_TATT_STILLING_TIL }) {
             stegService.resetSteg(behandling.id, StegType.VILKÅR)
+        } else if (erInitiellVurderingAvVilkår(behandling)) {
+            behandlingService.oppdaterStatusPåBehandling(behandling.id, BehandlingStatus.UTREDES)
+            opprettBehandlingsstatistikkTask(behandling)
         }
     }
 
+    private fun opprettBehandlingsstatistikkTask(behandling: Behandling) {
+        if (behandling.type != BehandlingType.BLANKETT) {
+            taskRepository.save(BehandlingsstatistikkTask.opprettTask(
```

Comment:
Hadde nesten kunnet haft flere metoder i `BehandlingsstatistikkTask`for de ulike hendelsene
Det gjør det litt tydligere om hva kontraktet er for de ulike typene
```
fun påbegynt(behandlingId: UUID): Task {
 return opprettTask(behandlingId, hendelse = Hendelse.PÅBEGYNT)
} 
```

Reply:
Enig, skriver om til ulike som tar inn ulike input (vi trenger forskjellige ting basert på hva som skjer)